modules/services/emacs.nix

```diff
@@ -10,23 +10,21 @@ let
   emacsVersion = getVersion emacsCfg.finalPackage;
 
   # Adapted from upstream emacs.desktop
-  clientDesktopItem = pkgs.makeDesktopItem rec {
-    name = "emacsclient";
```

Comment:
Wouldn't it be sufficient to change this name to something like `hmemacsclient`?

Reply:
Unfortunately, desktop file entries are keyed by [their filename](https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id). If we change the name of our desktop entry file, it does not override the entry which is shipped with Emacs, and two entries appear in the desktop database with the name "Emacs Client".

Reply:
Ah, right. Then perhaps setting the package priority to -1 so that it will overwrite the desktop file from the Emacs package?